src/store/socket.js

```diff
@@ -481,6 +485,41 @@ class LiveSession {
     }
   }
 
+  /**
+   * Publish a player pronouns update
+   * @param player
+   * @param value
+   */
+  sendPlayerPronouns({ player, value }) {
+    //send pronoun only for the seated player or storyteller
+    if (this._isSpectator && this._store.state.session.playerId !== player.id)
+      return;
+    const index = this._store.state.players.players.indexOf(player);
+    this._send("pronouns", { index, value, fromST: !this._isSpectator });
+  }
+
+  /**
+   * Update a pronouns based on incoming data. Player only.
+   * @param index
+   * @param value
+   * @param fromSt
+   * @private
+   */
+  _updatePlayerPronouns({ index, value, fromST }) {
+    const player = this._store.state.players.players[index];
+    if (
+      !player ||
+      (!fromST && this._store.state.session.playerId === player.id) ||
+      player.pronouns === value
+    )
```

Comment:
Instead of doing a `return` here, just flip the `if` around and only execute the next line if it is true.
```suggestion
    if (
      player &&
      (fromST || this._store.state.session.playerId !== player.id) &&
      player.pronouns !== value
    ) {
```